locust_tasks/locustfile.py

```diff
@@ -308,6 +318,10 @@ def report_failure(id, resp, task, failure_message, page_content):
 
     resp.failure(f'ID={id} UAC={task.case["uac"]} Status={resp.status_code}: {failure_message}')
     logger.error(f'ID={id} UAC={task.case["uac"]} Status={resp.status_code}: {failure_message}{error_detail}')
+    
+    # Slow down error reporting when things are going wrong (otherwise hundreds of errors are logged in just a few seconds)
+    time.sleep(5.0)
```

Comment:
please ensure that this 5 second sleep doesn't have adverse affect of recording errors (eg slows down the whole testing)

Reply:
I've hacked the code to force a sleep for one particular UAC. When one of the tasks does the sleep it has no effect on the progress of the other tasks.
Extended comment to note this fact, to hopefully stop people worrying about this in the future.